src/index.js

```diff
@@ -52,9 +51,10 @@ function Debug(options) {
   }
 
   var config = {
+    projectIdRequired: false,
     baseUrl: 'https://clouddebugger.googleapis.com/v2',
     scopes: [
-      // TODO: do we still need cloud-platform scope? 
+      // TODO: do we still need cloud-platform scope?
```

Comment:
According to the API explorer cloud-debugger now has its own scope:
`https://www.googleapis.com/auth/cloud_debugger`

It would be interesting to see if this new scope fulfills all our needs